tests/conftest.py

```diff
@@ -106,3 +107,16 @@ async def mcache(mcache_params):
     client = aiomcache.Client(**mcache_params)
     yield client
     await client.close()
+
+
+@pytest.yield_fixture
+async def mcache_pylibmc(mcache_params):
+    client = aiomcache.Client(
+        get_flag_handler=pylibmc_get_flag_handler,
+        set_flag_handler=pylibmc_set_flag_handler,
+        **mcache_params)
+    try:
+        yield client
+    except BaseException:
```

Comment:
Shouldn't this happen on a `finally`? Why would we not want to close it when there is no error?